ha/setup/ha_setup.py

```diff
@@ -291,8 +293,14 @@ def process(self):
                         self._confstore.set(f"{const.CLUSTER_CONFSTORE_NODES_KEY}/{node_name}")
                         Log.info(f"Added new node: {node_name}")
                         node_added = True
+                        nodes = self._confstore.get(f"{const.CLUSTER_CONFSTORE_NODES_KEY}")
+                        if node_added == True and len(nodes.keys()) == 2:
```

Comment:
node_added check is redundant, but ok.

Reply:
`if node_added == True`
Here only checking is done so not redundant but yes we already know value is true

Lets keep, this check may be helpful if code increases